stock_picking_report_valued/models/stock_move_line.py

```diff
@@ -55,7 +55,7 @@ def _compute_sale_order_line_fields(self):
             quantity = line._get_report_valued_quantity()
```

Comment:
if not line.sale_line: continue

you don't have to calculate this.
quantity = line._get_report_valued_quantity()